### PR TITLE
xmllint v4 examples

### PR DESCRIPTION
**Pull request by bwbohl**
applied command `find MEI_4.0 -type f -name '*.mei' -exec xmllint {} --format --output {} \;` with the default value of the XMLLINT_INDENT environment variable /two spaces ` `.

### CODE REVIEW

File: MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei [hunk 1]
```diff
@@ -59,21 +48,24 @@
...
-           <name>mei21To30.xsl</name>
-           <ptr target="https://github.com/music-encoding/encoding-tools/blob/main/mei21To30/mei21To30.xsl"/>
+          <name>mei21To30.xsl</name>
+          <ptr target="https://github.com/music-encoding/encoding-tools/blob/main/mei21To30/mei21To30.xsl"/>
         </application>
         <application version="v1.0_beta" xml:id="mei30To40">
           <name>mei30To40.xsl</name>
           <ptr target="https://github.com/music-encoding/encoding-tools/blob/main/mei30To40/mei30To40.xsl"/>
         </application>
+        <application version="0.0.1-alpha" xml:id="mei-lint_0.0.1-alpha">
```
Comment (musicEnfanthen) [inline MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei:58]:
```suggestion
        <application version="v0.0.1-alpha" xml:id="meiLint">
```

File: MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei [hunk 2]
```diff
@@ -59,21 +48,24 @@
...
+          <name>mei-lint.xsl</name>
+          <ptr target="https://github.com/music-encoding/encoding-tools/blob/main/linting/mei-lint.xsl"/>
```
Comment (musicEnfanthen) [inline MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei]:
```suggestion
        <application version="v0.0.1-alpha" xml:id="meiLint">
          <name>meiLint.xsl</name>
          <ptr target="https://github.com/music-encoding/encoding-tools/blob/main/meiLint/meiLint.xsl"/>
```

File: MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei [hunk 3]
```diff
@@ -259,6 +246,22 @@
...
+          <persName>Benjamin W. Bohl</persName>
+          <persName>Stefan Münnich</persName>
+        </respStmt>
+        <changeDesc>
+          <p><list>
+              <li>update xml-model schema references to v4.0.1</li>
+              <li>update meiversion attributes to 4.0.1</li>
+              <li>add missing responsibility statements to change elements</li>
+              <li>unify documentation of mei version upgrades (appInfo and revisionDesc)</li>
+               <li>linted with <ref target="#mei-lint_0.0.1-alpha">mei-lint.xsl</ref></li>
```
Comment (musicEnfanthen) [inline MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei]:
```suggestion
               <li>linted with <ref target="#meiLint">meiLint.xsl</ref></li>
```

File: MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei (hunk 2 above)
Comment (bwbohl) [reply MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei]: should be fixed in the new commits

Comment (bwbohl) [reply MEI_4.0/Header/Authority_data/Example_Authority_data_II.mei]: nah, missed the xml:id

**Review by musicEnfanthen — APPROVED** [review]

**Pull request merged**